hugo/content/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a free, open source, low latency, high quality voice chat application.
+
+## Source Code & Development
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
+
+# Who we are
+
+<!-- Title could also be: Developers, Team etc. -->
+
+<!-- improve -->
+
+We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.
+
+<!--
+Link to Github Team page:
+https://github.com/orgs/mumble-voip/people
+
+-->
```

Comment:
Probably a good idea :+1: 

The list is effectively outdated though (in terms of who is still active) since the majority of people listed there have not contributed to the project since I started working on it.

Reply:
> The list is effectively outdated though

Thats a problem.
If we list the Core team, the information should be fairly recent imo.

Reply:
I would simply not list that. I think the contributors page on GitHub contains all information that a user might want to know. Thus imo it should be enough to link to that :shrug:

Reply:
I disagree:
1. The Github Graph shows some inactive team members first.
2. One of the main intentions of an about page is to inform users, who the people behind the project are and this would be done best by listing the core team.

But if you don't want that, there is of course nothing I can do :rabbit:.

Reply:
Fair enough. In that case I think we could list
- @davidebeatrici - Core developer
- @Krzmbrzl - Core developer, GitHub Management
- @ZeroAbility - Release Management
- @Kissaki  - Community work
- @Natenom - Testing

I think as far as the core team and their current main tasks are concerned, that should be it (correct me if that's not the case).

Reply:
I assume you want no real names (though they can be found out easily anyway) ?
I will link to the Github profiles then.

> correct me if that's not the case

I have no clue :shrug:. But doesn't Kissaki & ZeroAbility do other things as well?

Reply:
If the names are available on the respective GitHub profile pages (which should be linked in any case), I think it is fine to also list the real names on the Website. If the real name is only contained in the commits, I would say that they should not be mentioned on the website.

> I have no clue shrug. But doesn't Kissaki & ZeroAbility do other things as well?

This comment was actually more directed towards the mentioned team members. I didn't expect you to know that :grin:

Reply:
> If the names are available on the respective GitHub profile pages (which should be linked in any case), I think it is fine to also list the real names on the Website

Ok, sounds good. Real names are more trustworthy.

> This comment was actually more directed towards the mentioned team members. I didn't expect you to know that

I already assumed that. But because they do respond very rarely (which is to some degree understandable if you got much to do and get many messages), it is probably on us to figure out the situation.

Reply:
> it is probably on us to figure out the situation.

Indeed. That's why I'd suggest overtaking the descriptions I provided until someone comes around and complains :D